packages/content-handler/src/websites/youtube-handler.ts

```diff
@@ -66,11 +106,12 @@ export class YoutubeHandler extends ContentHandler {
       <iframe width="${width}" height="${height}" src="https://www.youtube.com/embed/${videoId}" title="${title}" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
         <p><a href="${url}" target="_blank">${title}</a></p>
         <p itemscope="" itemprop="author" itemtype="http://schema.org/Person">By <a href="${oembed.author_url}" target="_blank">${authorName}</a></p>
+        <p class='omnivore-youtube-transcript'>
```

Comment:
One issue is what happens if this is null? I guess just an empty string wouldn't cause any issues here.

Reply:
Default empty string